kubectl-openebs/cli/command/get/volume_list.go

```diff
@@ -32,39 +31,28 @@ This command displays status of available zfs Volumes.
 If no volume ID is given, a list of all known volumes will be displayed.
 
 Usage: kubectl openebs cStor volume list [options]
-	`
-	namespace string
+`
 )
 
-const (
-	volumeStatusOK = "Running"
-)
-
-// NewCmdVolumesList displays status of OpenEBS Volume(s)
-func NewCmdVolumesList() *cobra.Command {
+// NewCmdGetVolume displays status of OpenEBS Volume(s)
+func NewCmdGetVolume() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "list",
-		Short: "Displays status information about Volume(s)",
-		Long:  volumesListCommandHelpText,
+		Use:     "volume",
+		Aliases: []string{"vol", "v", "volumes"},
+		Short:   "Displays status information about Volume(s)",
+		Long:    volumesListCommandHelpText,
 		Run: func(cmd *cobra.Command, args []string) {
 			util.CheckErr(RunVolumesList(cmd), util.Fatal)
 		},
 	}
-
-	cmd.Flags().StringVarP(&namespace, "namespace", "n", "openebs",
-		"namespace name, required if volume is not in the `default` namespace")
-
-	flag.CommandLine.Parse([]string{})
-
 	return cmd
 }
 
-//RunVolumesList fetchs & lists the volumes
+// RunVolumesList lists the volumes
 func RunVolumesList(cmd *cobra.Command) error {
-
-	client, err := client.NewK8sClient(namespace)
+	// TODO: Why is this working?
```

Comment:
I tried this by installing openebs on the `test` namespace and this still worked, I believe `""` used to be the k8s constant for default or all-namespaces(my memory is a lot hazy here).